docs/user/00-20-nfs.md

```diff
@@ -1,2 +1,61 @@
 # NFS
 
+## Overview
+
+The Cloud Manager module provides a Network File System (NFS) feature which allows you to create ReadWriteMany (RWX) volumes. RWX volumes can be used by multiple workloads.
```

Comment:
@ngrkajac please provide the sentence about the standard setup and the benefit of the NFS feature over the standard approach